### ISSUE TITLE
pure SVG widget

### ISSUE THREAD

**Issue by carloslfu** [cyclejs/cycle-dom#47]
I'm constructing svg widgets like:
```jsx
export default function workspace(responses) {
  const props$ = responses.props.get('*');

  const vtree$ = props$.map(props => {
    const privateProps = ['children', 'nested']
    const svgProps = _.pick(props, (value, key) => (privateProps.indexOf(key) == -1))
    return (
      <svg attributes={svgProps} style={{ border: '1px solid rgb(221, 221, 221)' }} >
        {props.children}
      </svg>
    )
  })

  return {
    DOM: vtree$,
  }
}
```
This displays in console: `Uncaught TypeError: rootElem.className.trim is not a function`, this is solved if I surround the widget DOM with a div, but then I cannot nest workspaces:
```jsx
<section>
  <workspace width={500} height={500} >
     <workspace width={100} height={100} >
     </workspace>
   </workspace>
 </section>
```

**Comment by staltz** [comment]
@carloslfu I released v5.3.1

**Comment by carloslfu** [comment]
Thanks @staltz
